### ISSUE TITLE
Piano tone

### ISSUE THREAD

**Issue by johnjacobkenny** [willismichael/virtual-playing-orchestra-ardour-template#1]
Hi Michael, thanks for the awesome compilation and the effort you've put it to make things seamless for use in Ardour. Truly saved tons of my time!

I'm not very impressed with the piano tone though. For me it sounds more like a guitar than a piano. Is there some setting that can be changed? I recently came across a free piano called salamander piano and it sounds much better to me. I was wondering if you had come across any other pianos and wanted to know if it was possible for you to update the existing piano.

**Comment by michaelwillis** [comment]
Hi Kenny! I'm glad you're finding it useful! I actually have tinkered around and changed the way I do orchestral arrangements in Ardour a lot since the last time I updated this, I've been considering making a new version of the template, but wasn't sure how much other people were actually using it. In particular, now I'm a lot happier using a different reverb plugin. To be honest I'm surprised that the template still works with the latest version of Virtual Playing Orchestra, since I thought that Paul Battersby had changed the names of some of the files or directories.

Regarding the piano, it's really a matter of taste; I tried the Salamander Piano several months ago and thought that the sound wasn't as crisp as I would like. On the other hand, I worked on a project with another musician who told me that he wasn't satisfied with the Maestro Concert Grand on its own, so he ended up mixing it with the grand piano from the Fluid Soundfont (fluid-soundfont-gs package on Debian, Ubuntu and relatives).

You can certainly customize the template to use the Salamander Piano. If you haven't already done this, the general idea is that you open your Ardour session (or just a new blank session from the template), launch QSampler or JSampler Fantasia to interact with the LinuxSampler plugin, find the row for the piano, edit it and replace the Maestro Concert Grand with the Salamander Piano, and then in Ardour use the `Session > Save Template` feature to either create your own template or save over this one.

Some other possibilities that you might try are the Little Lizard Piano from bandshed.net that you can find here: http://www.bandshed.net/sounds/sfz/ and the Piano in 162 from ivyaudio.com, found here: http://ivyaudio.com/Piano-in-162 ... The Little Lizard is simply a minimized version of the Salamander Piano, which results in the Ardour session being lighter weight on your RAM, and I don't actually remember much about Piano in 162 other than it was another free option that I tried a while ago.

If you're interested in trying an updated version of the Virtual Playing Orchestra Template, I can certainly audition some other options again and see what I think this time around; it's possible that my tastes will have changed since the last time I tried. In the very least, I want to make an update to take advantage of the latest Virtual Playing Orchestra, change the reverb plugin, and simplify the signal routing.

**Comment by michaelwillis** [comment]
Actually, now that I think about it, my dissatisfaction with the Salamander Piano may have been directly related to the previous reverb that I was using...  I'm going to try it out with my new reverb configuration and see what I think.

**Comment by johnjacobkenny** [comment]
Awesome. I also found a huge collection of instruments here https://sites.google.com/site/soundfonts4u/

You might have come across it before. I was wondering if anything from there can be used to improve the quality.

**Comment by johnjacobkenny** [comment]
Yes, indeed I had looked at the config file and figured out why the instrument wasn't working. I just renamed the folder so it would work correctly with the configured path.

**Comment by michaelwillis** [comment]
I just updated the template to use the brand new VPO 3 and a better reverb plugin. In my opinion, the entire thing sounds much better now. I did not replace the piano, @johnjacobkenny could you take a listen with the new reverb and let me know what you think now? You'll have to follow the instructions in the readme again due to the updated dependencies.

**Comment by johnjacobkenny** [comment]
Great! I'll give it a try and let you know. Thanks!

**Comment by johnjacobkenny** [comment]
I'm not sure if there's something wrong with the configuration on my end, but the piano still sounds very much "guitar like". On a side note, jSampler seems to just cause Ardour to crash when it is opened. It first takes some time to load but eventually Ardour crashes... Again I'm not sure if it is some misconfiguration.

Sorry for the delay, I was shifting home. Still getting things sorted.

**Comment by michaelwillis** [comment]
@johnjacobkenny If jSampler doesn't work for you, you can modify the file `/plugins/1811970/state2/linuxsampler`. Change the line with `/opt/maestro-concert-grand/maestro_concert_grand_v2.gig` such that it refers to whatever piano you want, and make sure that the format listed on the next line matches your sample (SFZ instead of GIG if you're using the Salamander piano).

**Comment by johnjacobkenny** [comment]
Oh great! Thanks

**Comment by bendavis78** [comment]
FYI the "Full Grand Piano" sound font from the _25 free piano soundfonts_ pack sounds amazing. I think it sounds better than salamander and most others I've heard. It also comes with a ton of variants with dark, bright, live, concert, etc.

https://www.flstudiomusic.com/2010/02/25-piano-soundfonts.html

**Comment by michaelwillis** [comment]
@bendavis78 Thanks! I wasn't aware of that collection. I've been thinking for a while that I should give this template a major overhaul; I've learned a lot since making it, and I know some ways to make it much more simple without compromising the stage presence of each section. I'm considering leaving out the piano, since there are a lot of different opinions about how people want a piano to sound.

**Comment by johnjacobkenny** [comment]
That would be great!

**Comment by johnjacobkenny** [comment]
I was having issues with Orchestra Performance Samples. Have you experienced anything?

**Comment by michaelwillis** [comment]
@johnjacobkenny The performance SFZs do not work with Linux Sampler. They use some opcodes that are not supported by Linuxsampler.

**Comment by johnjacobkenny** [comment]
Oh okay! Do you have any sample ardour sessions you could share so I could understand how you use this template :blush:
